docs/content/developers/testing-docs.md

```diff
@@ -10,9 +10,9 @@ The documentation is built and checked automatically with various [GitHub Action
 2. Click the pencil button on GitHub and follow the instructions to create your change.
 3. Save your changes and follow the prompts to create a PR.
 4. In the checks on your PR, click the “details” link by `docs/readthedocs.org:ddev` to browse the docs build created by your PR.
-5. Once the PR has run its checks, you'll see in the checks that there's a `docs/readthedocs.org:ddev` check with a link to the docs with your change in them. Follow the link to review what your change actually looks like:
-    <img width="861" alt="_docs__moodle_quickstart_by_ibrah3m_·_Pull_Request__4286_·_drud_ddev" src="https://user-images.githubusercontent.com/112444/196007331-7a1a3a0f-0fbe-41af-a50d-7f02b7d497f0.png">
-6. Take a look at the "Check docs" check to make sure that there were no linting or spellcheck errors.
+5. Once the PR has run its actions, you’ll see an item labeled `docs/readthedocs.org:ddev`. Click “Details” to review a docs build that includes your changes:
```

Comment:
They're actually called checks, but it doesn't matter.

Reply:
Hrm. If anything I thought [the right term was _workflow_](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions#the-components-of-github-actions) in this case—but my goal was to make the step easier to read. (_has run its checks_ (...) _in the checks_ is a little redundant and hard to step over, where the point is to get to the right “Details” link.)

Could just as well swap _actions_ for _workflows_ or _checks_!

Reply:
Not important enough.

Reply:
But... 
<img width="866" alt="_docs__Image_cleanup_by_mattstein_·_Pull_Request__4336_·_drud_ddev" src="https://user-images.githubusercontent.com/112444/197901836-20b1c988-2392-4598-bee4-9d5dc43a6243.png">

Reply:
Oh that’s devastating! Changing it to _checks_ since that’s what the person would see right there in the UI we’re pointing at. Good call.